libraries/AP_HAL_ChibiOS/hwdef/BeastH7v2/hwdef.dat

```diff
@@ -0,0 +1,46 @@
+# hw definition file for processing by chibios_pins.py
+# for iFlight Beast H7 v2 hardware, based on Beast H7 v1
+# thanks to betaflight for pin information
+
+include ../BeastH7/hwdef.dat
+undef APJ_BOARD_ID
+undef HAL_SERIAL3_PROTOCOL
+undef IMU
+undef BARO
+undef STM32_PWM_USE_ADVANCED
+undef PD15 #Buzzer
+undef PE9 PE11 # Motors
+undef HAL_DEFAULT_INS_FAST_SAMPLE
+
+# board ID for firmware load
+APJ_BOARD_ID 1056
+
+# V2 has different motor mapping
+PB4 TIM3_CH1 TIM3 PWM(2) GPIO(51)     # 3
```

Comment:
You need to add a BIDIR to one one these. Bi-dir applies in channel pairs - 1/2 or 3/4

Reply:
fixed